src/kinds.jl

```diff
@@ -880,6 +880,7 @@ const _kind_names =
         "tuple"
         "ref"
         "vect"
+        "parens"
```

Comment:
This is here because I wanted to use it in the test harness, and because I expect to use it in the main parser later on, as part of #88